pkg/proxy/iptables/proxier.go

```diff
@@ -1448,8 +1190,13 @@ func (proxier *Proxier) syncProxyRules() {
 		endpointChains = endpointChains[:0]
 		var endpointChain utiliptables.Chain
 		for _, ep := range proxier.endpointsMap[svcName] {
-			endpoints = append(endpoints, ep)
-			endpointChain = ep.endpointChain(svcNameString, protocol)
+			epInfo, ok := ep.(*endpointsInfo)
+			if !ok {
+				glog.Errorf("Failed to cast ep to endpointsInfo")
```

Comment:
Maybe include svcName and ep in the log message?

Reply:
Done.